examples/metrics/plot_itakura.py

```diff
@@ -0,0 +1,103 @@
+"""
+=====================
+Itakura parallelogram
+=====================
+
+This example explains how to set the `max_slope` parameter of the itakura
+parallelogram when computing the Dynamic Time Warping (DTW) with
+`method` == "itakura". The Itakura parallelogram is defined through a
+`max_slope` parameter which determines the slope of the steeper side. It is
+implemented in :func:`pyts.metrics.dtw.itakura_parallelogram`. The slope of the
+other side is set to "1 / max_slope" and eventually rescaled by the relative
+`n_timestamps_2` / `n_timestamps_2` if the time series have different lengths.
```

Comment:
Typo : `n_timestamps_2` / `n_timestamps_1`?
Maybe we should not mention how it is rescaled, since it is a bit more complicated:
```python
min_slope = 1 / max_slope
scale_max = (n_timestamps_2 - 1) / (n_timestamps_1 - 2)
max_slope *= scale_max
max_slope = max(1., max_slope)
scale_min = (n_timestamps_2 - 2) / (n_timestamps_1 - 1)
min_slope *= scale_min
min_slope = min(1., min_slope)
```